model/DeliveryAssemblyService.php

```diff
@@ -117,7 +115,9 @@ public function deleteInstance(core_kernel_classes_Resource $assembly)
      */
     protected function deleteDeliveryRuntime(core_kernel_classes_Resource $assembly)
     {
-        $this->getServiceManager()->get('taoDelivery/assignment')->onDelete($assembly);
+        /** @var GroupAssignment $deliveryAssignement */
+        $deliveryAssignement = $this->getServiceManager()->get('taoDelivery/assignment');
```

Comment:
Please use constant (`oat\taoDelivery\model\AssignmentService::CONFIG_ID`) here instead of string